fem/fespace.hpp

```diff
@@ -473,6 +495,19 @@ class FiniteElementSpace
    void GetTransferOperator(const FiniteElementSpace &coarse_fes,
                             OperatorHandle &T) const;
 
+   /** @brief Construct and return an Operator that can be used to transfer
+       GridFunction data from @a this FE space, defined on a fine mesh, to @a
+       coarse_fes, defined on a coarse mesh. */
+   /** This method returns a left inverse of the Operator returned by
+       GetTransferOperator().
+
```

Comment:
Should we clarify that we assume that the operator (R) returned by `GetTransferOperator()` is full column rank? In particular we assume that the LOR space has dimension that is at least as big as that of the HO space.

Reply:
I agree it would be good to make this explicit, perhaps even issuing a warning (error?) if this not the case.